### PR TITLE
Add CD deployment workflow for dApp on Testnet

### PR DESCRIPTION
**Pull request by david-iov**
### Continuous Deployment for dApp deployed on S3+CF

Create a TAG with the suffix '-beta' against the updated branch to run with and that would trigger the workflow to deploy to TESTNET

### CODE REVIEW

**Review by antomor — CHANGES_REQUESTED** [review]
@david-iov thank you for these changes. However that would imply that each time that we create a new release, the dapp is deployed to testnet. I'm not sure that this is exactly what we want. How do we plan to manage the mainnet release? 
I think it would be better if we can release to testnet only if the release tag is a beta tag. WDYT?

File: .github/workflows/deploy-testnet.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+name: Testnet CD
+
+on:
+  push:
+    tags:
+      - '**-beta'
```
Comment (antomor) [inline .github/workflows/deploy-testnet.yml]: Question: should be this something like `**-beta*`? It should work with any beta tag, so tags like `1.0.0-beta.x` should be taken into account.
I'm not sure if with the filter `**-beta` we take into account `x.y.z-beta` only.

Comment (david-iov) [reply .github/workflows/deploy-testnet.yml]: Okay, I will update that.

Comment (david-iov) [reply .github/workflows/deploy-testnet.yml]: Updated

**Review by rafaiovlabs — APPROVED** [review]

**Review by david-iov — COMMENTED** [review]
LGTM

**Review by antomor — APPROVED** [review]
@david-iov Thank you for the changes, LGTM!

**Pull request merged**